howto/customize_paths.md

```diff
@@ -0,0 +1,33 @@
+## Customize paths
+
+The project is setup to use specific paths to read the original weights and save checkpoints etc.
+
+For all scripts, you can run
+
+```shell
+python script.py -h
+```
+
+to get a list of available options. For instance, here's how you would modify the checkpoint dir:
+
+```shell
+python scripts/convert_checkpoint.py --checkpoint_dir "/srv/data/checkpoints/foo"
```

Comment:
if we could remove the /srv prefix here, I think it looks a bit odd :))
Just `/data/...` would be enough